cs/client.py

```diff
@@ -54,6 +54,8 @@ def transform(params):
             continue
         if isinstance(value, string_type):
             continue
+        if key == "userdata" and isinstance(value, bytes):
```

Comment:
Hi @vojtechcima!

Could you add `binary_type = str` in the `if PY2:` clause, `binary_type = bytes` in the `else` clause (at the top of the file) and check for `isinstance(value, binary_type)` instead? This ensures consistent behavior in Python 2 and 3.

Thank you!